File: Sheets/Swift/string.swift
// Split String
let s="leetcode"
let ss="leet"
print(s[ss.startIndex..<ss.endIndex])
print(s[ss.endIndex..<s.endIndex])


// String from array
let array = ["Andrew", "Ben", "John", "Paul", "Peter", "Laura"]
let joined = array.joined(separator: ", ")
print("String from Array: \(joined)")

// Array From String
var str = "super String with awesome TExt"
var arr = str.components(separatedBy: " ")
print("Array from string: \(arr)")

File: Sheets/Python/string.py
# String from array
array = ["Andrew", "Ben", "John", "Paul", "Peter", "Laura"]
joined = ", ".join(array)
print("String from Array: "+joined) # String from Array: Andrew, Ben, John, Paul, Peter, Laura

print(60*"-")

# Array from string
someStr = "super String with awesome TExt"
arr = someStr.split(" ")
print("Array from string: " + str(arr)) # Array from string: ['super', 'String', 'with', 'awesome', 'TExt']

print(60*"-")


# Split String
s="leetcode"
ss="leet"
print(s[:len(ss)]) # leet
print(s[len(ss):]) # code


s = "123"
i = 1
tra = int(s[i:i+2])  # This is int("23") => 23
